source/extensions/filters/http/lua/lua_filter.h

```diff
@@ -156,10 +159,31 @@ class StreamHandleWrapper : public Filters::Common::Lua::BaseLuaObject<StreamHan
    * @param 2 (table): A table of HTTP headers. :method, :path, and :authority must be defined.
    * @param 3 (string): Body. Can be nil.
    * @param 4 (int): Timeout in milliseconds for the call.
+   * @param 5 (bool): Flag. If true, filter continues without waiting for HTTP response from upstream service.
```

Comment:
this is optional right? probably want to explicitly specify the default behavior